redux-kotlin-threadsafe/build.gradle

```diff
@@ -72,7 +70,6 @@ kotlin {
         }
         jsMain {
             dependencies {
```

Comment:
Nice clean up, thanks!  Lets remove the empty dependencies {}

Reply:
Cleaned up